styles/tree-view.less

```diff
@@ -59,3 +59,7 @@
     }
   }
 }
+
+.name.dragenter {
```

Comment:
This selector is in the global scope, so this will affect anything with the classes `.name` and `.dragenter` – can you change this to at least be scoped inside `.tree-view`? Ideally the selector should probably be `.tree-view .entry .name`, but nesting it inside the existing `.tree-view` block would be best.